RecoEcal/EgammaClusterProducers/python/particleFlowSuperClusterECALHLT_cfi.py

```diff
@@ -0,0 +1,77 @@
+import FWCore.ParameterSet.Config as cms
+
+particleFlowSuperClusterECALHLT = cms.EDProducer("PFECALSuperClusterProducer",
```

Comment:
does this really make it suitable for HLT or is this really a reco without vtx?
For the latter, perhaps NoVtx is a better suffix than HLT.

I see that many parameters repeat particleFlowSuperClusterECALMustache. Some explicit clone-replace seems more appropriate.

Reply:
Yes, this is the same configuration used at HLT, except for the input collections (`offlineBeamspot` vs `hltOnlineBeamspot`, `ecalRecHit` vs `hltEcalRecHit`, *etc*.), see for example https://github.com/cms-sw/cmssw/blob/acf9623a7368ca0bf1200b7411291b4dbf9c2483/HLTrigger/Configuration/python/HLT_GRun_cff.py#L13521-L13565.

> I see that many parameters repeat particleFlowSuperClusterECALMustache. Some explicit clone-replace seems more appropriate.

I tend to agree - I left it fully expanded since this is what Box and Mustache did.
If ECAL people don't have a strong preference, I'll change it to
```python
from RecoEcal.EgammaClusterProducers.particleFlowSuperClusterECALMustache_cfi import particleFlowSuperClusterECALMustache as _particleFlowSuperClusterECALMustache

# "Mustache" clustering
particleFlowSuperClusterECALHLT = _particleFlowSuperClusterECALMustache.clone()

# HLT regression setup
particleFlowSuperClusterECALHLT.regressionConfig = _particleFlowSuperClusterECALMustache.regressionConfig.clone(
    isHLT = cms.bool(True),
    eRecHitThreshold = cms.double(1.),
    regressionKeyEB  = cms.string('pfscecal_EBCorrection_online'),
    uncertaintyKeyEB = cms.string('pfscecal_EBUncertainty_online'),
    regressionKeyEE  = cms.string('pfscecal_EECorrection_online'),
    uncertaintyKeyEE = cms.string('pfscecal_EEUncertainty_online'),
    vertexCollection = cms.InputTag()
)
```

Reply:
Hi @fwyzard,
We have no objections from the ECAL sides. The fully expanded forms looks good to us.

Thanks,
Rajdeep